src/aws-cpp-sdk-core/include/aws/core/utils/logging/LogMacros.h

```diff
@@ -119,76 +72,16 @@
             { \
                 Aws::OStringStream logStream; \
                 logStream << streamExpression; \
-                logSystem->LogStream( logLevel, tag, logStream ); \
```

Comment:
There was an issue here:
 `logLevel` is changed now to (a macro parameter) `level`